include/mega/node.h

```diff
@@ -209,53 +209,62 @@ struct MEGA_API Node : public NodeCore, FileFingerprint
     NodeCounter subnodeCounts() const;
 
     // parent
-    Node* parent;
+    Node* parent = nullptr;
 
     // children
     node_list children;
 
     // own position in parent's children
-    node_list::iterator child_it;
+    node_list::iterator child_it{};
```

Comment:
Do we need {} here?  Since it's an object of a class, if it has a default constructor then it would be called anyway.

Reply:
I suppose we don't need it. Will remove